### PR TITLE
Change UI links

### PR DESCRIPTION
**Pull request by ItaloKnox**
**Please don't merge this yet.** The files aren't in production yet!

I'll merge it after I make sure the script is working as intended.

### CODE REVIEW

**Comment by ItaloKnox** [comment]
The files are there already, but I dunno how to rerun the checks to make sure everything is working. Pls help

File: installData.json [hunk 1]
```diff
@@ -262,9 +262,10 @@
 					],
 					"fileOverrides" : [
 						{"name": "movie",   "os": ["mac", "linux"], "url":"https://07th-mod.com/rikachama/video/Meakashi-Movie_UNIX.7z"},
-						{"name": "ui",      "os": ["windows"],      "unity": "5.5.3p3", "url":"https://07th-mod.com/rikachama/ui/Meakashi-UI_steam_win.7z"},
-						{"name": "ui",      "os": ["windows"],      "unity": "5.5.3p1", "url":"https://07th-mod.com/rikachama/ui/Meakashi-UI_gog_win.7z"},
-						{"name": "ui",      "os": ["mac", "linux"], "unity": "5.5.3p3", "url":"https://07th-mod.com/rikachama/ui/Meakashi-UI_steam_unix.7z"}
+						{"name": "ui",      "os": ["windows"],      "unity": "5.5.3p3", "url":"https://07th-mod.com/ui.php?chapter=meakashi&os=win&unity=5.5.3p3"},
+						{"name": "ui",      "os": ["windows"],      "unity": "5.5.3p1", "url":"https://07th-mod.com/ui.php?chapter=meakashi&os=win&unity=5.5.3p1"},
+						{"name": "ui",      "os": ["mac", "linux"], "unity": "5.5.3p3", "url":"https://07th-mod.com/ui.php?chapter=meakashi&os=unix&unity=5.5.3p3"}
```
Comment (drojf) [inline installData.json]: The build failed cause you're missing a comma here (and possibly other things as well, didn't look through it in detail)

Comment (drojf) [inline installData.json]: The build failed cause you're missing a comma here (and possibly other things as well, didn't look through it in detail)

**Comment by drojf** [comment]
er please ignore what I just wrote

**Comment by drojf** [comment]
I just fixed the missing comma - if you use visual studio code to edit the .json it should pickup any json errors automatically (I can't remember if I installed a linter or not, but I think that feature comes by default)

**Comment by drojf** [comment]
@ItaloKnox  it seems to have failed on this link: https://07th-mod.com/ui.php?chapter=tatarigoroshi&os=win&unity=5.3.4p1

which gives a 404 for me when I try to open it

**Comment by ItaloKnox** [comment]
I swear I fixed the link.

**Comment by drojf** [comment]
I think it's just the usual thing where the github link times out which is causing it to fail this time. I'll run it again

**Pull request merged**